### ISSUE TITLE
Unable to load crypto

### ISSUE THREAD

**Issue by Piyush5767** [stealjs/stealjs#112]
I have tried to load the crypto library by importing nodeRequire whereas in steal js and in donejs framework getting an error 

![crypto](https://user-images.githubusercontent.com/8464646/57266618-bfd7ea00-704a-11e9-9d4c-a382070a6ef9.PNG)

I have tried to import nodeRequire module via
import nodeRequire from "@node-require";

and also tried to load a crypto module via system: var crypto =  System._nodeRequire('crypto');

In donejs application getting error: 

**Error: Error loading "crypto" at file:C:/xampp/htdocs/hp/crypto.js
Error loading "crypto" from "hp@1.8.0#index.stache!done-autorender@0.8.0#autorender" at file:C:/xampp/htdocs/hp/src/index.stache
ENOENT: no such file or directory, open 'C:\xampp\htdocs\hp\crypto.js'**

**Comment by Piyush5767** [comment]
Also tried to load it via
var nodeRequire = require("@loader")._nodeRequire;
nodeRequire("crypto");
 but still getting same error: Error loading "crypto"

**Comment by matthewp** [comment]
That's really bizarre. `System._nodeRequire` should be the literally Node.js require functions. it should be able to import any builtins.  What do you get when you console.log that function? `console.log(System._nodeRequire)`?

**Comment by Piyush5767** [comment]
Yes, the above one works by using nodeRequire by using the system but about other libraries for example xml2js I am getting an error in node WebKit when I include that library
 var xml2js = require('xml2js');

Gives me:
 Error: Cannot find module 'xml2js'

**Comment by matthewp** [comment]
I was able to get xml2js to load like this: https://gist.github.com/matthewp/982f8e34dcf8ba1db53984c4e5fa0e9a

What are you doing differently?
